tests/test_project.py

```diff
@@ -106,13 +105,21 @@ def test_root_directory(self):
     def test_workspace_directory(self):
         assert self._tmp_wd == self.project.workspace()
 
+    @pytest.mark.filterwarnings(
+        "ignore:Modifying the project configuration after project initialization "
+        "is deprecated"
+    )
```

Comment:
Please separate all the unrelated filterwarnings into a separate commit.

Reply:
Resolved by #603.